### DISCUSSION TITLE
What should i do?

### DISCUSSION THREAD

**Discussion by JackieDisgrace** [Discord-OTP-Forcer/Discord-OTP-Forcer#121] [Community Support]
As you can see, i have the ChromeDriver that (almost) match with my Chrome Ver.
But it says it only supports 122
What the heck should i do? Download a Chrome Ver. 122? Please help me
I'm almost done, i need to enter in my Alt account, my main got disable for a mistake by the stupid Trust & Safety System, i can't lose my work, i need to get on the Alt
![imagem_2024-06-19_192029339](https://github.com/Derpitron/Discord-OTP-Forcer/assets/173312244/6ac879b7-f241-4140-97b0-38cd6757cd10)
![imagem_2024-06-19_192059527](https://github.com/Derpitron/Discord-OTP-Forcer/assets/173312244/879c129f-f1fa-4b5b-a8ef-8106dfe43278)

**Comment by Inbydev** [comment]
@JackieDisgrace could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

**Reply by JackieDisgrace** [reply]
Trying but "No module named 'stackprinter'" keeps happening, I know I have to run "pip install -r requirements.txt" but where? I run Python and I paste that command there but it only get me invalid syntax

**Reply by MicXDev** [reply]
You need to run from a command prompt rather than python itself as it's technically not part of python.

**Reply by Inbydev** [reply]
@JackieDisgrace you are using Windows, so did you try the scripts that are on the project?

scripts/windows/setup.bat

and later

scripts/windows/start.bat

**Reply by MicXDev** [reply]
> Trying but "No module named 'stackprinter'" keeps happening, I know I have to run "pip install -r requirements.txt" but where? I run Python and I paste that command there but it only get me invalid syntax

@JackieDisgrace It may be help trying it again, last night I made commits which fixed some bugs with pip not being found for installing the requirements. Download the latest commit and run `setup.cmd` (it's now `.cmd` rather than `.bat`). Also, try running `uninstall_requirements.cmd` as well first to ensure you get fresh packages.

**Reply by Inbydev** [reply]
follow the tutorial here @JackieDisgrace:

https://discord-otp-forcer.codeberg.page/en/

with the latest commit:

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta
